tratto/src/main/java/star/tratto/util/javaparser/JDoctorUtils.java

```diff
@@ -0,0 +1,210 @@
+package star.tratto.util.javaparser;
+
+import star.tratto.identifiers.ConditionPrimitiveType;
+import star.tratto.identifiers.JPType;
+
+import com.github.javaparser.ast.body.CallableDeclaration;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.Parameter;
+import com.github.javaparser.ast.body.TypeDeclaration;
+import com.github.javaparser.ast.type.TypeParameter;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * This class manages the differences between JDoctor and JavaParser
+ * representations of variables, as well as other utilities for both type
+ * representations.
+ */
+public class JDoctorUtils {
+    /**
+     * The method converts the list of JDoctor type names {@code jDoctorTypeNames} into a list of JavaParser type names.
+     * For example, the JDoctor type name <span>[D</span> represents a list of doubles, and the corresponding type name in
+     * JavaParser is <span>double[]</span>. The method apply these conversions from JDoctor type names to JavaParser type
+     * names.
+     *
+     * @param jDoctorTypeNames the list of JDoctor type names to convert
+     * @return the list of the corresponding JavaParser type names
+     */
+    public static List<String> convertJDoctorConditionTypeNames2JavaParserTypeNames(
+            List<String> jDoctorTypeNames
+    ) {
+        return jDoctorTypeNames
+                .stream()
+                .map(JDoctorUtils::convertConditionParameterType)
+                .collect(Collectors.toList());
+    }
+
+    public static String convertConditionToArrayType(String arrayType) {
+        int howMany = 0;
+        for (char c : arrayType.toCharArray()) {
+            if (c == '[') {
+                howMany++;
+            } else {
+                break;
+            }
+        }
+        return convertToArrayType(arrayType.replaceAll("^\\[+",""), howMany);
+    }
+
+    public static String convertToArrayType(String arrayType, int howMany) {
```

Comment:
What is the output format?  What is the input format?  Why does this return a string rather than a two-element data structure with a toString method?  (Maybe that wouldn't be better, maybe it would.)